web-registry/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -86,7 +86,7 @@ function ProjectDetails(): JSX.Element {
 
   const onChainProjectId = isOnChainId
     ? projectId
-    : dataByHandle?.projectByHandle?.onChainId;
+    : dataByHandle?.projectByHandle?.onChainId ?? undefined;
```

Comment:
TS compiler found null as a possibility here, so explicitly falling back to `undefined`.